onnx/defs/generator/defs.cc

```diff
@@ -159,7 +159,8 @@ ONNX_OPERATOR_SET_SCHEMA(
         .Output(0, "output", "Output tensor, same shape as input tensor T1.", "T2")
         .TypeConstraint(
             "T1",
-            {"tensor(float16)",
+            {"tensor(bfloat16)",
```

Comment:
Will this fit into any of the existing "named sets of types"? Or, should we create a new one? Looks like this should apply to all types that have a zero and one. (It makes sense for complex too, but we don't support complex in any meaningful operation anyway, so ignoring that seems fine.)

Reply:
I mean the various types defined [here](https://github.com/onnx/onnx/blob/4128a09009aa67622c6308c82fe4199813a71682/onnx/defs/schema.h#L587)

Reply:
This list as-is doesn't fit any because of complex and fp8. we can add them here or just keep this list as-is. I was unsure and played safe here

Reply:
changed to OpSchema::all_tensor_types_ir10(). That will include complex, but that should be fine